static/src/stylesheets/module/experiments/_update-account.scss

```diff
@@ -0,0 +1,18 @@
+.site-message--membership-action-required {
+    color: $garnett-neutral-5;
+    background: $garnett-neutral-1;
+    &:after {
+        @include multiline(3, $garnett-neutral-7, top);
+        content: '';
+        display: block;
+        height: 4px * 3; /*height of multiline (4px) times the number of lines (4)*/
```

Comment:
Why not use $gs-baseline which is 12px?

Reply:
I have run into needing the multiline height a bunch of times too and in every other repo i've ended up making a `multiline-height($lines)` function that multiline itself also uses, would that be useful/desirable in frontend?

Reply:
The height value here does not correspond with the comment. Which is correct? 😝

Reply:
The final (4) in the comment should be a (3). I'm not using $gs-baseline because the 12 is coming from the multiline and not the baseline.